admin/tests/integration/components/campaigns/update_test.js

```diff
@@ -0,0 +1,129 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { click, fillIn, render } from '@ember/test-helpers';
+import sinon from 'sinon';
+import hbs from 'htmlbars-inline-precompile';
+import EmberObject from '@ember/object';
+
+module('Integration | Component | Campaigns | Update', function(hooks) {
+  setupRenderingTest(hooks);
+
+  let toggleEditMode;
+  let update;
+  let campaign;
+
+  hooks.beforeEach(function() {
+    update = sinon.stub();
+    toggleEditMode = sinon.stub();
+    const save = sinon.stub();
+    campaign = EmberObject.create({ title: 'Ceci est un titre', name: 'Ceci est un nom', save });
+
+    this.set('update', update);
+    this.set('toggleEditMode', toggleEditMode);
+    this.set('campaign', campaign);
+  });
```

Comment:
[suggestion]
Normalement on peut directement écrire

```js
  hooks.beforeEach(function() {
    this.update = sinon.stub();
    this.toggleEditMode = sinon.stub();
    this.campaign = EmberObject.create({
      title: 'Ceci est un titre',
      name: 'Ceci est un nom',
      save: sinon.stub(),
    });
  });
```

Et directement utiliser `this.update`, `this.campaign`... par la suite